reo/nested_inputs.py

```diff
@@ -87,8 +87,8 @@ def list_of_str(input):
     "timeout_seconds": {
       "type": "int",
       "min": 1,
-      "max": 420,
-      "default": 420,
+      "max": 600,
```

Comment:
has this longer timeout been agreed upon for the webtool?

Reply:
We haven't discussed it; for now I've reverted this edit in commit b3be5a07b71294b6fb9b76338d75cb5abf88d6fc

That said, it would be good to revisit this after some more testing to see if this needs to be moved up.